test/pool.test.ts

```diff
@@ -80,12 +107,17 @@ async function setReceiver(
   } else {
     receivers.set(address, weight);
   }
-  expectReceivers(pool, receivers);
+  await expectReceivers(pool, receivers);
 }
 
 async function getAllReceivers(pool: Pool): Promise<Map<string, number>> {
   const receivers = await pool.getAllReceivers();
-  return new Map(receivers.map(({receiver, weight}) => [receiver, weight]));
+  return new Map(
+    receivers.map(({receiver, weight}: {receiver: string; weight: number}) => [
```

Comment:
Do we need an explicit type annotation? This seemed to be ok before.

Reply:
I was sure that it was producing warnings, but you're right, it's unnecessary, fixed